lib/ain-evm/src/evm.rs

```diff
@@ -367,19 +347,37 @@ impl EVMServices {
         })
     }
 
-    pub fn verify_tx_fees(&self, tx: &str, use_context: bool) -> Result<(), Box<dyn Error>> {
+    pub fn finalize_block(&self, queue_id: u64) -> Result<(), Box<dyn Error>> {
+        let BlockData { block, receipts } = match self.core.tx_queues.get_block_data(queue_id) {
+            Some(block_data) => block_data,
+            None => return Err(anyhow!("finalize block failed, no block in queue_id").into()),
+        };
```

Comment:
Can use `.ok_or_else(|| anyhow!("finalize block failed, no block in queue_id"))?` instead of match here

Reply:
Resolved in commit 2701ef00e

Reply:
Resolved in commit 2701ef00edbe1818d